apps/charterafrica/src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -0,0 +1,72 @@
+import { Card, CardContent, CardMedia, Typography } from "@mui/material";
+import React from "react";
+
+import { neutral, secondary } from "@/charterafrica/colors";
+
+const OpportunityCard = React.forwardRef(function OpportunityCard(props, ref) {
+  const { opportunity } = props;
```

Comment:
```suggestion
  const { title, description, deadline, link } = props;
```

Why the need to full opportunity object only to destructure it?